extensions/hibernate-search-orm-elasticsearch/deployment/pom.xml

```diff
@@ -25,6 +25,10 @@
             <groupId>io.quarkus</groupId>
             <artifactId>quarkus-elasticsearch-rest-client-common-deployment</artifactId>
         </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-vertx-http-deployment</artifactId>
```

Comment:
Just to be sure, it's done this way in the other extensions? We only add the deployment one?

Reply:
Well it depends of the extension. We need the runtime one. In some extensions, the `-deployment` is not imported but `quarkus-vertx-http` and `vertx-web` dependencies are imported as optional, this is the case in liquibase, cache, flyway extensions. But in some other extensions both `quarkus-vertx-http` (runtime and deployment) are imported, this is the case in smallrye-openapi.
It looks like if the `quarkus-vertx-http` module was not present before, it is declared without the `-deployment` as optional. I will do this.